charts/localstack/test-values.yaml

```
@@ -20,6 +20,12 @@ mountDind:
   forceTLS: true
 
 
+ingressHost: chart-example.local
+ingress:
+  enabled: true
+  hosts:
+    - host: "{{ .Values.ingressHost }}"
```

This now runs when creating the template, but unfortunately not when applying it to an actual cluster:
```bash
helm install ls-helm-demo ./charts/localstack/ -f ./charts/localstack/test-values.yaml                                                                                                                                 11:28:36
Error: INSTALLATION FAILED: Ingress.extensions "ls-helm-demo-localstack" is invalid: spec.rules[0].http.paths: Required value
```

```suggestion
ingressHost: chart-example.local
ingress:
  enabled: true
  hosts:
    - host: "{{ .Values.ingressHost }}"
      paths:
      - path: /
        pathType: ImplementationSpecific
```

---

fixed and pushed